src/table/TableBodyWrapper.jsx

```diff
@@ -12,12 +12,22 @@ const useStyles = makeStyles({
     fontSize: props.fontSize,
     color: props.fontColor,
   }),
+  hoverTableRow: (props) => ({
+    '&&:hover': {
+      backgroundColor: props.hoverBackGroundColor,
+      '& td': {
+        color: props.hoverFontColor,
+      },
+    },
+  }),
 });
 
-export default function TableBodyWrapper({ tableData, constraints, selectionsAPI, layout, theme }) {
+export default function TableBodyWrapper({ tableData, constraints, selectionsAPI, layout, theme, cellLockedImage }) {
   const { rows, columns } = tableData;
-  const classes = useStyles(getBodyStyle(layout, theme));
+  const hoverEffect = layout.components?.[0]?.hoverEffect;
+  const classes = useStyles(getBodyStyle(layout, theme, hoverEffect));
   const getColumnRenderers = (selectionsEnabled) => tableData.columns.map((c) => getCellRenderer(c, selectionsEnabled));
+  const [selectionsEnabled, setSelectionsEnabled] = useState(false);
```

Comment:
Why do you need an extra state to store this? it's already a property on `cellState`.

Reply:
I missing that. Will fix it. Thanks.